### ISSUE TITLE
Support for RNA004?

### ISSUE THREAD

**Issue by Delayed-Gitification** [Psy-Fer/deeplexicon#27]
Hi,

Just wondering if you have trained any models on barcodes for RNA004? This would be super useful!

Many thanks
Oscar

**Comment by Psy-Fer** [comment]
Hello Oscar,

We did plan on doing this much earlier, however, we were not given early access to the kits/flowcells, and along with the announcement that nanopore were going to release a barcoding method for dRNA, we put this on a very low-priority (see slow5/pod5 for why)

Fortunately, the original group of researchers who created the deeplexicon method, now make up multiple other research groups. I have discussed this briefly with one of my collaborators and they are willing to do the new barcode designs (many more than 4 this time) and sequence them to generate the training data. Our deep learning specialist is also keen to do the training. If all of this happens, I will update the tooling to integrate it all into a fancy new release.

Unfortunately, I don't have a timeline for you, again, with nanopore barcoding methods 2 weeks or 2 years away (who knows, they certainly don't) it's not exactly an enticing project to throw our time and effort behind in place of other projects.

So tl;dr: We will, but it's not our top priority.

As a little side note, we can do some other cool things with the release of many many more barcodes. So there is at least the enticement of some cool down stream method development that can be done.

Cheers,
James

**Comment by Delayed-Gitification** [comment]
Thanks James. Yes, I can see why this is a difficult project to prioritise given the situation with nanopore. I actually contacted a rep to ask when barcoding is likely to be released by them. I doubt they will tell me but I'll let you know if they do!

Best
Oscar

**Comment by enovoa** [comment]
Dear Oscar,  we have been training a different algorithm for RNA004 runs. 
Feel free to email me directly to get early access to alternative software for demuxing RNA004 runs if you're still interested in this. Thanks! Eva

**Comment by jibrantahir** [comment]
Hi I am catching up on this old question- is there a possibility to get early access to alternative software for demuxing RNA004 with possibly more barcodes- 96? Also another question- if the barcode sequence is present in the sequences from both ends, why cant we just use this information to demultiplex with conventional methods - like cutadapt, many thanks

**Comment by enovoa** [comment]
Dear @jibrantahir as mentioned above, please feel free to  mail me directly to get early access to alternative software for demuxing RNA004 runs. Thanks! Eva

**Comment by enovoa** [comment]
Update: Following requests from people that had seen this issue and emailed me about this, demultiplexing RNA004 datasets for the same set of barcodes that DeePlexiCon was using, can now be done with [SeqTagger](https://github.com/novoalab/SeqTagger). The work is published [here](https://genome.cshlp.org/content/early/2025/01/29/gr.279290.124.full.pdf). It also supports POD5. Thanks!

**Comment by Psy-Fer** [comment]
Thanks @enovoa
